turbinia/workers/hindsight.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for executing hindsight."""
+
+from __future__ import unicode_literals
+
+import os
+
+from turbinia.evidence import TextFile
+from turbinia.workers import TurbiniaTask
+
+
+class HindsightTask(TurbiniaTask):
+  """Task to execute hindsight."""
+
+  def __init__(self):
+    super(HindsightTask, self).__init__()
+
+  def run(self, evidence, result):
+    """Task to execute hindsight."""
+
+    # Create the new Evidence object that will be generated by this Task.
+    output_evidence = TextFile()
+
+    # Create a path that we can write the new file to.
+    output_file_path = os.path.join(self.output_dir, 'hindsight_report')
```

Comment:
Just one small thing here:  Can we set the output file extension to `evidence.output_format`?

Reply:
Sorry I saw your comment earlier but hindsight accepts the output path without the extension and adding that can cause double extensions.

Reply:
Ah, I see.  Ok, thanks!  LG then, I'll merge now.